gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 # plugins
-kotlin = "1.9.24"
```

Comment:
Is this downgrade intentional?

Reply:
I was getting some warnings but will try putting it back.

Reply:
That's Gradle complaining that its own embedded version is ever so slightly different. We can either:

- ignore this, as long as the minor version is correct I've never had any issues
- see if there's a new Gradle release with the higher version
- do the revert